deployment/integration.yaml

```diff
@@ -0,0 +1,10 @@
+route:
+  enabled: true
+  host: cloud-portal-integration.apps.cloudscale-lpg-2.appuio.cloud
+image:
+  registry: image-registry.openshift-image-registry.svc:5000
+  pullPolicy: Always
+portal:
+  config:
+    issuer: https://id.appuio.cloud/auth/realms/appuio-cloud
+    clientId: appuio-cloud-api
```

Comment:
@bastjan I'm not sure about the client ID. the current value doesn't exist yet. I'm guessing this needs to be a new client? Or is it the one where the Control API is running on? :smile:

Reply:
So, as it stands out, we can choose/create an arbitrary new client id. However, in Keycloak we cannot configure the "valid redirect URI" with a wildcard pattern like `https://portal-pr-*-appuio-control-api-preview.apps.cloudscale-lpg-2.appuio.cloud/*` where it would accept every PR.

Looking for options...

Reply:
I went with the approach where we register redirect URI in Keycloak client dynamically via Keycloak API. Sorry about the notification noise